### PR TITLE
[DOC]: Removed a redundant 'The'

### PR DESCRIPTION
**Pull request by dSupertramp**
Hi, this is my first pull request in an open source project.

I know it's not fair to write this kind of description here, but it's just a way for you to leave a message.

Advice is welcome, thank you!

### CODE REVIEW

**Comment by tacaswell** [comment]
Thanks!  We appreciate contributions of any size and kind.  Documenattion fixes from people new to the project are particularly valuable as (at least I) tend to get "word blind" due to familiarity.

While I agree that the wording now is not quite right, but I do no think dropping the "the" is the right fix.  If were to drop the "Python scripts", the sentence would read

> Matplotlib can be used in Python and IPython shell, web application servers, and 

which also does not read quite right.   Maybe 

> Matplotlib can be used in Python scripts, the Python or IPython shells, web application servers, and 

would read better?

**Comment by dSupertramp** [comment]
In my opinion, it sounds better without "the": maybe I'm wrong

**Comment by dSupertramp** [comment]
> Thanks!  We appreciate contributions of any size and kind.  Documenattion fixes from people new to the project are particularly valuable as (at least I) tend to get "word blind" due to familiarity.
> 
> While I agree that the wording now is not quite right, but I do no think dropping the "the" is the right fix.  If were to drop the "Python scripts", the sentence would read
> 
> > Matplotlib can be used in Python and IPython shell, web application servers, and 
> 
> which also does not read quite right.   Maybe 
> 
> > Matplotlib can be used in Python scripts, the Python or IPython shells, web application servers, and 
> 
> would read better?

Honestly, in my opinion, it sounds better without "the": maybe I'm wrong

**Comment by tacaswell** [comment]
> Honestly, in my opinion, it sounds better without "the": 

Fair. I still do not agree, but not strongly enough to block merging,  

Our policy on documentation PRs is one person with commit rights agreeing it is an improvement to merge so I leave this to someone else to merge.

----

The test failures here are unrelated and should be ignored (the py37 in linux is the coverage fork issue and the OSX failures are (I suspect) a gs version change issue).

File: README.rst
Comment (jklymak) [inline README.rst:54]: I'd also go with:
```suggestion
in Python scripts, the Python or IPython shells, web application servers, and
```

Comment (timhoffm) [inline README.rst:54]: The 'the' is correct in the present formulation. The long enumeration mainly consisting of general plurals may add to the confusion. If you boil that down to

> Matplotlib can be used in the Python and IPython shell.
vs.
> Matplotlib can be used in Python and IPython shell.

Nevertheless, it's a bit awkward in the enumeration.

I suggest

```suggestion
in Python scripts, Python/IPython shells, web application servers, and
```

Comment (dSupertramp) [reply README.rst:54]: This suggestion is much better, i like it.

Thanks a lot for the answer.

Since I am a newbie, how should I proceed in this case?
Can I go with "commit suggestions"?

Any other suggestion is welcome!

Comment (timhoffm) [reply README.rst]: Yes, since you're the author you can accept commit suggestions. In this case I've done that now for you, so that CI can already run.

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Thanks @dSupertramp, and congratulations on your first contribution to Matplotlib! We hope to hear from you again.

**Comment by dSupertramp** [comment]
> Thanks @dSupertramp, and congratulations on your first contribution to Matplotlib! We hope to hear from you again.

Thanks for the opportunity!

**Pull request merged**
